soc/arm/nordic_nrf/nrf53/soc.c

```diff
@@ -133,6 +133,7 @@ static bool nrf53_anomaly_160_check(void)
 
 	return true;
 }
+#endif /* CONFIG_SOC_NRF53_ANOMALY_160_WORKAROUND */
```

Comment:
Just a detail.
This causes that `z_arm_on_enter_cpu_idle` is defined always, even if neither of those two workarounds uses it. That hook might be potentially needed for some other purpose, so I think it would be good to wrap its definition in
```
#if defined(CONFIG_SOC_NRF53_ANOMALY_160_WORKAROUND) || \
	(defined(CONFIG_SOC_NRF53_RTC_PRETICK) && defined(CONFIG_SOC_NRF5340_CPUNET))
```

Reply:
done